src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -390,21 +390,35 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
 #Region "Buffer helpers"
 
+        Private Function NextAre(chars As String) As Boolean
+            Debug.Assert(Not String.IsNullOrEmpty(chars))
+            Dim n = chars.Length-1
```

Comment:
The "-1" doesn't seem right to me. Consider that this should be equivalent to ```NextAre(0, chars)```
The "-1" should be on the upper bound of the For loop on line 397